docs/rfc/rfc-9.adoc

```diff
@@ -0,0 +1,145 @@
+:toc: macro
+
+= RFC 9: Bitcoin Ordinals
+
+:icons: font
+:numbered:
+toc::[]
+
+== Background
+
+As of 2023-02-23, Bitcoin Ordinals are 🔥 HOT 🔥. Here is a
+https://insights.glassnode.com/ordinal-theory-and-the-rise-of-inscriptions/[decent
+explainer].
+
+image:assets/inscriptions-over-time.png[]
+
+There is not a good way to get those Ordinals over to Ethereum currently, where
+Ordinal owners can use the more mature NFT platforms. We could be that way!
+
+Here is an https://ordinals.com/inscription/cfa453dbf693641b8f0c6b83cb733e71832606fc26415c36c70744953d6f6161i0[example ordinal].
+
+=== Current Functionality
+
+Ordinals, at a high level, are single satoshis with metadata stuck into their
+witness field (called Inscribing). Each satoshi has an immutable age, which
+effectively gives them serial numbers.
+
+These Inscribed Satoshis (called Inscriptions) can be transferred like normal
+BTC. The sender's wallet software needs to be careful to not merge the
+Inscriptions with "normal" UTXOs, and needs to be careful to not accidentally
+send the Inscription as part of a payment. The
+https://github.com/casey/ord[casey/ord] repo provides the primatives for the
+handling of Ordinals on BTC.
+
+A good mental model here is thinking about Inscriptions like
+https://www.apmex.com/product/117942/1932-s-washington-quarter-choice-au[special
+edition quarters]. The the linked quarter trades for ~$350, but is also legal
+tender worth $0.25. An owner of such a quarter probably does not want to use it
+to pay for parking. Inscriptions work the same way.
+
+Since Ordinals are still BTC, we can use most of the same tBTCv2 ideas for
+custodying Ordinals: rotating wallets, P2SH addresses, fraud proofs,
+redemptions, etc, with some slight modifications.
+
+== Proposal
+
+=== Overview
+
+We build a BTC <> Ethereum Ordinal bridge!
+
+We create 51-of-100 threshold signing groups using the same process that we use
+for tBTCv2. To deposit to these signing groups, we use the same P2SH process
+including a refund that we use for tBTCv2. They pay a <<Fees,fee>>. We
+optimistically mint using the same Minter & Guardian process we already have. In
+the background, we sweep to disable the refund. Later, someone brings back the
+NFT and a <<Fees,fee>>. We burn the NFT and send them their Ordinal.
+
+=== Depositing Details
+
+Depositing works essentially the same way that it does in tBTCv2 with a few key
+differences.
+
+Once a deposit is revealed, it is **extremely** important that the wallet does
+not sweep them into the same UTXO; that would be destructive. We still need to
+sweep to disable the refundfootnote:[Whether or not the sweep can be performed
```

Comment:
I believe it can, with a multi-output sweep tx.

Reply:
Yup - [double checked](https://bitcoin.stackexchange.com/questions/35686/how-do-you-send-to-multiple-addresses-by-paying-a-one-time-fee-which-wallet-sup) it

Reply:
Clarified in [284318](https://github.com/keep-network/tbtc-v2/pull/561/commits/284318bc2beb0b9a25c0ad88fd4c022a1869117f)